interfaces/builtin/modem_manager.go

```diff
@@ -173,6 +173,9 @@ dbus (receive, send)
 
 # Allow to determine whether a tty device is a serial port or not.
 @{PROC}/tty/drivers r,
+
+# allow communicating with mbim-proxy server
+unix (connect, receive, send) type=stream peer=(addr="@{mbim,qmi}-proxy"),
```

Comment:
This adds both the mbim-proxy and qmi-proxy addresses - can the comment please be updated to also mention qmi-proxy for completeness? Perhaps even something along the lines of:

```
# allow communicating with the mbim and qmi proxy servers, which provide
# support for talking to WWAN modems and devices which speak the Mobile 
# Interface Broadband Model (MBIM) and Qualcomm MSM Interface (QMI) 
# protocols respectively
```

So that it is clear what and why this access is granted?

Reply:
The comment was modified to be clearer.